techmatters/flex-plugins
# ch: update @twilio/flex-ui to 2.3.x

Primary reviewer: @stephenhand 

## Description

- Package Upgrade: Update @twilio/flex-ui to 2.3.x

### Checklist
- [N/A] New tests added
- [N/A] Feature flags added
- [N/A] Strings are localized
- [N/A] Tested for chat contacts
- [N/A] Tested for call contacts

### Related Issues
Fixes CHI-1712

### Verification steps

- Run `npm install`
- Run `npm run dev`
- Verify that everything still works as expected
